### DISCUSSION TITLE
Tasmota 12.4

### DISCUSSION THREAD

**Discussion by ptommson** [benzino77/tasmocompiler#336] [General]
Hi,
I'm new to this but was wondering if you could please tell me if it's possible to add Tasmota 12.4 to the versions in the compile?
The new Matter support looks interesting!
Thanks

**Comment by benzino77** [comment]
I will - I need to find time for that. Maybe at the end of the week...

**Reply by ptommson** [reply]
Thanks :)

**Reply by benzino77** [reply]
New TasmoCompiler version is available which includes Tasmota v12.4.0.

**Reply by ptommson** [reply]
Awesome thanks! I will give it a try
